chart/kubeapps/README.md

```diff
@@ -1,3 +1,5 @@
+<!--- app-name: Kubeapps -->
```

Comment:
Unfortunately, we have to remove that part. The docs will fail otherwise.
However, since this file is symlinked, netlify is not being aware of the change, thus not rendering a preview.

```suggestion
```